docs/spec/api.md

```diff
@@ -117,6 +117,9 @@ specification to correspond with the versions enumerated here.
 			<li>Deleting a manifest by tag has been deprecated.</li>
 			<li>Specified `Docker-Content-Digest` header for appropriate entities.</li>
 			<li>Added error code for unsupported operations.</li>
+			<li>Added capability of doing streaming upload to PATCH blob upload.</li>
```

Comment:
@dmcgowan Sounds acceptable, although many of those changes happened pre-2.0.0. Feel free to reorganize as you see fit. Perhaps, this is where we capture the difference in the reference implementation.